File: YazilimGelistiriciDotNetHomeWork/source/repos/KampHomeWorkJune/Day5Homework4InterfaceAbstract/Abstract/BaseCustomerManager.cs
using Day5Homework4InterfaceAbstract.Concrete;
using System;
using System.Collections.Generic;
using System.Text;

namespace Day5Homework4InterfaceAbstract.Abstract
{
    public abstract class BaseCustomerManager : ICustomerService
    {
        public virtual void Save(Customer customer)
        {
            Console.WriteLine("Saved to database : " + customer.FirstName);
        }

    }
}

File: JavaReactKampHomeWork/homeworkJavaSE1.8June/src/day4Homework2InterfaceAbstract/abstracts/BaseCustomerManager.java
package day4Homework2InterfaceAbstract.abstracts;

import day4Homework2InterfaceAbstract.entities.Customer;

public abstract class BaseCustomerManager implements CustomerService{
	
	@Override
	public void save(Customer customer) {
		System.out.println("Saved to database : " + customer.getFirstName());
	}

}
